AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/AwsCryptographyMaterialProvidersOperations.dfy

```diff
@@ -270,6 +291,7 @@ module AwsCryptographyMaterialProvidersOperations refines AbstractAwsCryptograph
     var cmc;
 
     if input.cache.Some? {
+      var _ := CheckCache(input.cache.value, input.ttlSeconds);
```

Comment:
I think that all of these need to be `:-`

Reply:
Whoops.